doc/sphinx/eduro.rst

```diff
@@ -0,0 +1,35 @@
+Robot Eduro
+===========
+
+Eduro was designed and created by Jan Roubicek back in 2007 as EDUcational
+RObot for the university. The electronics was manufactured by Tomas Roubicek,
+and it includes CAN network modules using CANopen protocol, custom made stepper
+motor drivers with encoders (SMAC) and power management.
```

Comment:
What is SMAC? https://en.wikipedia.org/wiki/SMAC is not helping 😟. There are really encoders? Hmm.